idseq_pipeline/commands/host_filtering_functions.py

```diff
@@ -24,11 +24,11 @@
 VERSION_OUT = 'versions.json'
 
 # arguments from environment variables
-INPUT_BUCKET = os.environ.get('INPUT_BUCKET')
-FILE_TYPE = os.environ.get('FILE_TYPE')
-OUTPUT_BUCKET = os.environ.get('OUTPUT_BUCKET')
-STAR_GENOME = os.environ.get('STAR_GENOME', 's3://czbiohub-infectious-disease/references/human/STAR_genome.tar')
-BOWTIE2_GENOME = os.environ.get('BOWTIE2_GENOME', 's3://czbiohub-infectious-disease/references/human/bowtie2_genome.tar')
```

Comment:
Removing the defaults here also removes some documentation value.   Worth adding a comment to indicate what an example path looks like.